<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadVideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Background Substraction</string>
    </property>
    <widget class="QPushButton" name="BackgroundSubstraction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Background Substraction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Optical Flow</string>
    </property>
    <widget class="QPushButton" name="Preprocessing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Preprocessing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Vediotracking">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Vedio tracking</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>3. PCA</string>
    </property>
    <widget class="QPushButton" name="DimensionReduction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Dimension Reduction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>4. MNIST Classifier Using VGG19</string>
    </property>
    <widget class="QPushButton" name="ShowModelStructure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowAccuracyandLoss">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2. Show Accuracy and Loss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Reset">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4. Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Predict">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Predict</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict :</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="title">
      <string>Draw</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>671</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>5. ResNet50</string>
    </property>
    <widget class="QPushButton" name="ResnetLoadImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowImages">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5.1 Show Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowComprasion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5.5 Show Comprasion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowModelStructureQ5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5.2  Show Model Structure</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>481</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Inference">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5.4 Inference</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
